### DISCUSSION TITLE
Adding custom text field to umbraco form setting

### DISCUSSION THREAD

**Discussion by troshan** [umbraco/Umbraco.Forms.Issues#1517] [Ideas]
There are many tutorials and documentation available about adding additional attributes to field types, such as  [ https://docs.umbraco.com/umbraco-forms/developer/extending/adding-a-fieldtype](https://docs.umbraco.com/umbraco-forms/developer/extending/adding-a-fieldtype) but I couldn't find any documentation about adding a custom setting field to the Umbraco form setting window, as this feature is not available. Please refer to the attached image
![Form_setting](https://github.com/umbraco/Umbraco.Forms.Issues/assets/6112822/15bb73d9-2b45-44b3-899c-8cf16a8397d8)
It would be great if this customization feature were added to Umbraco Forms, as it would extend the functionality of forms and allow for further customization.

**Comment by tedlindholm** [comment]
Yes, this would be very useful, feels simple to implement 😀

**Comment by TvGessel** [comment]
This is a duplicate of discussion #1632. I'll close this one so we can centralize the feedback there.
